java/src/Ice/src/main/java/com/zeroc/Ice/OperationMode.java

```diff
@@ -15,6 +15,8 @@
 
 package com.zeroc.Ice;
```

Comment:
This file should be generated code

Reply:
Alright, I'll remove it, and add the correct metadata to the slice file then. Just wanted to make sure before I started scraping things I shouldn't.